src/roles/vsc-upgrade-deploy/tasks/main.yml

```diff
@@ -113,4 +113,15 @@
       that: "uptime_after_reboot.stdout < uptime_before_reboot.stdout"
       fail_msg: "System uptime after reboot step suggests there were issues during reboot. Check log for errors."
 
+  - name: Get primary config filename
+    set_fact:
+      vsc_config_filename: "{{ lookup('file', backup_folder + '/primary_config_filename') }}"
+
+  - name: Copy VSC config file to the VSC
+    delegate_to: localhost
+    shell: >-
+      sshpass -p'{{ vsc_password | default(vsc_default_password) }}' scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null
+      {{ proxy_conf | default('') }} {{ backup_folder }}/{{ vsc_config_filename }} {{ vsc_username | default(vsc_default_username) }}@{{ mgmt_ip }}:
+    #no_log: "{{ lookup('env', 'METROAE_NO_LOG') or 'true' }}"
```

Comment:
We want no_log to be set now. Please uncomment this

Reply:
ok sure.